src/main/java/com/fewlaps/quitnowemailsuggester/EmailSuggester.java

```diff
@@ -114,7 +114,7 @@ private String fixDomain(String email, String badDomain, String goodDomain) thro
         return email;
     }
 
-    private String fixDomainAndTld(String email, String badDomainAndTld, String goodDomainAndTld) {
+    private static String fixDomainAndTld(String email, String badDomainAndTld, String goodDomainAndTld) {
```

Comment:
@m-ezzat hi! Having static methods makes testing hard, 'cause it's difficult to mock it using mockito. What do you think about this? Thanks!